packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -151,18 +478,24 @@ describe('<TracePage>', () => {
 
       it('calls props.focusUiFindMatches with props.trace.data and uiFind when props.trace.data is present', () => {
         const uiFind = 'test ui find';
-        wrapper.setProps({ uiFind });
-        wrapper.find(TracePageHeader).prop('focusUiFindMatches')();
+        wrapper.rerender(<TracePage {...defaultProps} uiFind={uiFind} />);
+        fireEvent.click(screen.getByTestId('focus-matches'));
         expect(defaultProps.focusUiFindMatches).toHaveBeenCalledWith(defaultProps.trace.data, uiFind);
         expect(trackFocusSpy).toHaveBeenCalledTimes(1);
       });
 
       it('handles when props.trace.data is absent', () => {
-        const propFn = wrapper.find(TracePageHeader).prop('focusUiFindMatches');
-        wrapper.setProps({ trace: {} });
-        propFn();
-        expect(defaultProps.focusUiFindMatches).not.toHaveBeenCalled();
-        expect(trackFocusSpy).not.toHaveBeenCalled();
+        const emptyTraceData = transformTraceData({
+          traceID: 'empty-trace',
+          spans: [],
+          processes: {},
+          warnings: [],
+        });
+        const emptyTrace = { data: emptyTraceData, state: fetchedState.DONE };
+        wrapper.rerender(<TracePage {...defaultProps} trace={emptyTrace} />);
+        fireEvent.click(screen.getByTestId('focus-matches'));
+        expect(defaultProps.focusUiFindMatches).toHaveBeenCalledWith(emptyTraceData, undefined);
+        expect(trackFocusSpy).toHaveBeenCalledTimes(1);
```

Comment:
these conditions now seem to be opposite of what was in the previous version of the test